packages/slice-machine/lib/models/common/Environment.ts

```diff
@@ -13,17 +13,22 @@ export interface UpdateVersionInfo {
   updateAvailable: boolean;
 }
 
-export default interface Environment {
+export default interface BackendEnvironment extends FrontEndEnvironment {
```

Comment:
OPT: I think it should be a named export so that you import explicitly either the Backend or Frontend Env

Reply:
I removed the inherit between the two types, I think the idea is to have two separates envs right ?